package.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from '@sanity/pkg-utils'
+
+export default defineConfig({
+  legacyExports: true,
+  dist: 'dist',
+  tsconfig: 'tsconfig.dist.json',
+
+  // Remove this block to enable strict export validation
```

Comment:
Can/should we?

Reply:
No - I had to add this in order to fix the build.